math/SuiteSparse/Portfile

```diff
@@ -188,7 +188,7 @@ subport SuiteSparse_SPQR {
 }
 
 subport SuiteSparse_SLIP_LU {
-    version                 2.0.9
+    version                 1.0.2
```

Comment:
shy is this version set back to an earlier version? If this is the reason for the "epoch" change then only set it here to "1" in this subport.

Reply:
I see from the comment that there is no actual change here besides the version number. I would suggest not to change anything here and just update it when upstream does.

Reply:
@reneeotten Just to make sure we're on the same page: The version number in MacPorts was incorrect (copy-paste error from `SuiteSparse_SPQR`). The real version is 1.0.2, and it did not change between SuiteSparse 5.13 and the previous SuiteSparse available in MacPorts.

Upstream will never update this version. Instead, SuiteSparse 6.0 and later replaces SLIP_LU with SPEX.

Before I go ahead, can you please confirm that you still want me to leave the (incorrect) version alone?

Reply:
(In the meantime I pushed the change that I understand you are asking for, so we can make progress on this.)